### ISSUE TITLE
Remember GitHub authentication on Windows 10

### ISSUE THREAD

**Issue by Fydon** [microsoft/vscode-pull-request-github#2759]
I see that it is possible to remember the GitHub authentication on Linux (#2562). How would I do this on Windows?

There are no errors on the GitHub Authentication output, only info. It does state "Adding session found in keychain".

VS Code details:
```
Version: 1.57.0-insider (system setup)
Commit: 217f1a2f60a197807de78af362ecb832100fda96
Date: 2021-06-02T06:36:06.263Z
Electron: 12.0.9
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19042
```

**Comment by alexr00** [comment]
@Fydon this should just work on Windows. A few questions:
- How often are you being prompted to sign in?
- Can you share a screen shot of where you're seeing the sign in prompt?
- How are you launching vscode?
- What version of the GitHub Pull Requests and Issues extension do you have installed?

**Comment by Fydon** [comment]
Thank you for responding.

* Every time I restart my PC
* The same prompts from #2562. I could add some screenshots when I next restart if you'd prefer?
* I'm launching from the Windows Start Menu
* I'm using version v0.26.0 of GitHub Pull Requests and Issues

**Comment by alexr00** [comment]
@Fydon just to further clarify: if you just close vscode and re-open it (no PC restart), you do not get the prompt?

**Comment by Fydon** [comment]
@alexr00 Yes I found that a bit strange too, so I restarted and it appears to be remembering the authentication now.

I've been logging in every day for some time so I'm not sure what changed. Something I noticed is that I wasn't able to login through Accounts before (where Settings Sync is, image shown below). I would go through the Authorization process, but it would continue to show the 1 by Accounts. I would only be able to log in through the extension directly and it would only log in in one window. Now I can both log in through Accounts and logging in in one window updates all windows. Maybe this is because I'm using the Insider build of VS Code. Sorry to have bothered you with this.

![image](https://user-images.githubusercontent.com/3404761/120701395-aebc6a00-c4aa-11eb-879d-9092e0f674f5.png)
